packages/edit-site/src/components/dataviews/field-types.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import Media from '../../components/media';
+
+export const coreFieldTypes = {
+	string: {
+		view:
+			( getValue ) =>
+			( { item } ) =>
+				getValue( { item } ),
+	},
+	date: {
+		view: ( { item, getValue } ) => <time>{ getValue( { item } ) }</time>,
+	},
+	image: {
```

Comment:
This works for WP images now, where we pass the attachment id. Later we could change name here and also add a field type that gets the `src` or more props and renders just an `img` tag.